src/scripts/cfo.zig

```diff
@@ -383,12 +417,6 @@ fn run_fuzzers_prepare_repository(shell: *Shell, target: union(enum) {
             break :commit try run_fuzzers_commit_info(shell);
         },
     };
-    log.info("fuzzing commit={s} timestamp={d}", .{ commit.sha, commit.timestamp });
-    // Strictly speaking, we don't need to build the code here, but doing this makes timing
-    // information on seeds more precise.
-    shell.zig("build -Drelease build_fuzz", .{}) catch |err| {
-        log.warn("failed to build: {}", .{err});
-    };
```

Comment:
Well, lets build beforehand doesn't really work given that we have _two_ build configs for vopr, which we might or might not need (depending on PR labels). I _could_ solve this by carefully compiling before starting the timer, but I am not sure it is worth it, and I want to tidy CLI interface and build.zig before that, so that I don't need even more duplication here.